.github/workflows/release.yml

```diff
@@ -82,16 +85,20 @@ jobs:
     steps:
       - name: Repository 🗂️
         uses: actions/checkout@v4
+        with:
+          ref: ${{ env.ENVIRONMENT }}
 
       - name: Deploy ⚡
         uses: ./.github/actions/mssql
         with:
+          branch: ${{ env.ENVIRONMENT }}
```

Comment:
setting a branch to an environment seems a little odd here, wouldn't environment be a deployed environment and branch be a branch name of something?